uncrustify.cfg

```diff
@@ -40,12 +42,14 @@ nl_namespace_brace		= add
 mod_full_brace_if		= remove
 mod_full_brace_if_chain	= true
 mod_full_brace_do		= remove
-mod_full_brace_while		= remove
+mod_full_brace_while	= remove
```

Comment:
@evsh:  This change makes the tab alignment even less consistent…

Reply:
Same as above.